### DISCUSSION TITLE
Omit e. g. 'language' works only for full bib export, not selected items?

### DISCUSSION THREAD

**Discussion by 192933488S** [retorquere/zotero-better-bibtex#2055] [Q&A]
Can anyone verify please:

Reproduction steps:

1. Have latest Zotero (+ lib content), latest BBT
2. Set Exports -> Fields -> Omit: 'language'
3. Export a selected item using right click -> Export item
4. Compare with full export of bib File -> Export library
5. Result: Selected item export does not omit 'language'

Would be nice if it works for selected items too.

Regards

**Comment by retorquere** [comment]
Can't help you without a support log ID.

**Reply by 192933488S** [reply]
What do you mean by that? A log or issue ticket?

**Reply by retorquere** [reply]
It's best to open an issue, you will get an explanation on how to generate a log ID.

**Comment by 192933488S** [comment]
Done! Thanks in advance for checking.
